From ccf37db6e6b983bafcd7a6a7abd7e066026b388f Mon Sep 17 00:00:00 2001
From: "Jason A. Donenfeld" <Jason@zx2c4.com>
Date: Fri, 10 Jun 2022 22:28:48 +0200
Subject: build: include compat.h for all files

54b2075 ("build: only include compat.h for if_wg.c and fix build with an
obj directory") scoped compat.h to if_wg.c, which broke building on
12.3.

Signed-off-by: Jason A. Donenfeld <Jason@zx2c4.com>
---
 src/Makefile | 2 +-

--- Makefile
+++ Makefile
@@ -6,6 +6,6 @@ SRCS= opt_inet.h opt_inet6.h device_if.h bus_if.h ifdi_if.h
 
 SRCS+= if_wg.c wg_noise.c wg_cookie.c crypto.c
 
-CFLAGS.if_wg.c+= -include ${.CURDIR}/compat.h
+CFLAGS+= -include ${.CURDIR}/compat.h
 
 .include <bsd.kmod.mk>
